### PR TITLE
Different null comparison since unity might be funny about it

### PR DESCRIPTION
**Pull request by Bod9001**
yeah, not too sure but I know that nulls are a bit funny in unity

### CODE REVIEW

File: UnityProject/Assets/Scripts/ScriptableObjects/SingletonScriptableObject.cs [hunk 1]
```diff
@@ -18,21 +18,21 @@ public static T Instance
 		{
 			get
 			{
-				if (!_instance && SOs.Instance != null)
+				if (_instance == null && (SOs.Instance == null) == false)
 				{
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/ScriptableObjects/SingletonScriptableObject.cs:22]: != Was fine here, I think. If anything the == false makes it harder to read than !=

Comment (Bod9001) [reply UnityProject/Assets/Scripts/ScriptableObjects/SingletonScriptableObject.cs:22]: is unity okay with that?

Comment (corp-0) [reply UnityProject/Assets/Scripts/ScriptableObjects/SingletonScriptableObject.cs:22]: With != ? I don't see why not, it should be using the same equality comparison

Comment (Bod9001) [reply UnityProject/Assets/Scripts/ScriptableObjects/SingletonScriptableObject.cs:22]: ok cool

**Review by ThatDan123 — APPROVED** [review]

**Review by corp-0 — APPROVED** [review]

**Pull request merged**